cmd/explaintest/r/explain_easy.result

```diff
@@ -320,6 +320,50 @@ Projection_11	10000.00	root	9_aux_0
       │   └─TableScan_27	10.00	cop	table:t, keep order:false, stats:pseudo
       └─TableReader_33	10.00	root	data:TableScan_32
         └─TableScan_32	10.00	cop	table:t1, range: decided by [s.c], keep order:false, stats:pseudo
+insert into t values(1, 1, 1), (2, 2 ,2), (3, 3, 3), (4, 3, 4),(5,3,5);
+analyze table t;
+explain select t.c in (select count(*) from t s, t t1 where s.b = t.a and s.b = 3 and s.a = t1.a) from t;
+id	count	task	operator info
+Projection_11	5.00	root	9_aux_0
+└─Apply_13	5.00	root	left outer semi join, inner:StreamAgg_20, other cond:eq(test.t.c, 7_col_0)
+  ├─TableReader_15	5.00	root	data:TableScan_14
+  │ └─TableScan_14	5.00	cop	table:t, range:[-inf,+inf], keep order:false
+  └─StreamAgg_20	1.00	root	funcs:count(1)
+    └─IndexJoin_49	2.40	root	inner join, inner:TableReader_48, outer key:s.a, inner key:t1.a
+      ├─IndexReader_41	2.40	root	index:Selection_40
+      │ └─Selection_40	2.40	cop	eq(3, test.t.a)
+      │   └─IndexScan_39	3.00	cop	table:s, index:b, range:[3,3], keep order:false
+      └─TableReader_48	4.00	root	data:Selection_47
+        └─Selection_47	4.00	cop	eq(3, test.t.a)
```

Comment:
For reviewers: the row count of `Selection_47` and `TableReader_48` is wrong, but this should be a separate issue I would fix in another PR later.